Push subscription state and enablement edits (#3197)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/push/users_and_subscriptions.md

- Foreground push tokens
- Background push tokens

Foreground push tokens provide the ability to send regular push notifications to the foreground of a user's device. Background, or silent, push tokens are assigned to all devices that have a brand’s app downloaded, regardless of that particular device has opted-in to receive push notifications from that brand. Background push tokens allow brands to send silent push notifications, which are push notifications that intentionally won’t be displayed to devices in order to support key functionalities like uninstall tracking.

When a user profile has a valid foreground push token associated with an app, Braze considers the user "push registered" for the given app. Braze, then, provides a specific segmentation filter `Push enabled for App` to help identify these users.

Note that `Push enabled for App` only considers the presence of a valid a foreground push token for the given app. However, the more generic `Push Enabled` filter considers the presence of any valid foreground push token as well as the push subscription state having a value of “subscribed” or “opted-in”.

### Check user's push enablement status

> On the **Engagement** tab in a user's profile, you will see **Push Registered For** followed by an app name. If no app information exists for that device, you will see  two dashes (**&#45;&#45;**). There will be an entry for every device that belongs to the user.

Reviewer: ```suggestion
On the **Engagement** tab in a user's profile, you will see **Push Registered For** followed by an app name. If no app information exists for that device, you will see two dashes (**&#45;&#45;**). There will be an entry for every device that belongs to the user.
```

If the device entry's app name is prefixed by `Foreground:`, the app is authorized to receive both foreground push notifications (visible to the user) and background push notifications (not visible to the user) on that device.
![Push Changelog][2]{: style="float:right;max-width:40%;margin-left:15px;margin-top:10px;"}

> On the other hand, if the device entry's app name is prefixed by `Background:`, the app is only authorized to receive [background push]({{site.baseurl}}/user_guide/message_building_by_channel/push/types/#background-push-notifications) and can not display user-visible notifications on that device. This usually indicates the user has disabled notifications for the app on that device.

Reviewer: ```suggestion
On the other hand, if the device entry's app name is prefixed by `Background:`, the app is only authorized to receive [background push]({{site.baseurl}}/user_guide/message_building_by_channel/push/types/#background-push-notifications) and cannot display user-visible notifications on that device. This usually indicates the user has disabled notifications for the app on that device.
```
